prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -239,7 +239,7 @@ void rxr_pkt_handle_ctrl_sent(struct rxr_ep *rxr_ep, struct rxr_pkt_entry *pkt_e
 }
 
 ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_entry,
-			       int ctrl_type, bool inject)
+			       int ctrl_type, bool inject, uint64_t flags)
```

Comment:
I think we need a comment at the beginning at the function, explain all parameters of this function

Reply:
Added, also add the comment for `rxr_pkt_post_ctrl`